src-electron/rest/user-data.js

```diff
@@ -50,7 +50,7 @@ function httpGetSessionKeyValues(db) {
     let sessionId = request.zapSessionId
     let sessionKeyValues = await querySession.getAllSessionKeyValues(
       db,
-      sessionId
+      sessionId,
```

Comment:
What is the comma doing here?

Reply:
This was a problem with the formatter changes that is being fixed with this PR..